### PR TITLE
Eye of the storm

### PR DESCRIPTION
**Pull request by 59blargedy**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by kaincenteno** [comment]
the npc_list.sql looks weird, maybe it needed to be rebased? way too many additions/deletions

File: scripts/zones/Chamber_of_Oracles/npcs/Armoury_Crate.lua
Comment (cocosolos) [inline scripts/zones/Chamber_of_Oracles/npcs/Armoury_Crate.lua]: Amount should just be 24k, loot handling will split it automatically.
https://youtu.be/TlvThE0sM14?t=950

File: sql/bcnm_battlefield.sql
Comment (cocosolos) [inline sql/bcnm_battlefield.sql]: Duplicate entries?

Comment (59blargedy) [reply sql/bcnm_battlefield.sql]: yep :( thanks for catching

File: scripts/zones/Chamber_of_Oracles/npcs/Armoury_Crate.lua
Comment (59blargedy) [reply scripts/zones/Chamber_of_Oracles/npcs/Armoury_Crate.lua]: got it, wasn't sure how that split worked and thought i would need some getplayercount shenanigans i wasnt prepared yet to do. much easier, thank you!

**Comment by 59blargedy** [comment]
> 
> 
> the npc_list.sql looks weird, maybe it needed to be rebased? way too many additions/deletions

I'm not sure what i did or how to fix it. Do you know?

**Comment by 59blargedy** [comment]
@kaincenteno i figured out what i did wrong and how to fix it, sorry about that :(

File: scripts/zones/Chamber_of_Oracles/mobs/Radiant_Wyvern.lua
Comment (KnowOne134) [inline scripts/zones/Chamber_of_Oracles/mobs/Radiant_Wyvern.lua]: monstertpmoves isnt needed as a require in all the mob scripts

**Review by KnowOne134 — COMMENTED** [review]
wiki says they have significant Movement Speed, heavy sleep resistance, and a special ability right before each die

File: scripts/zones/Chamber_of_Oracles/mobs/Radiant_Wyvern.lua
Comment (59blargedy) [reply scripts/zones/Chamber_of_Oracles/mobs/Radiant_Wyvern.lua]: they dont have a special ability before death, wiki is just referring to their unique breath attacks. will look at other alterations - wyverns already have boosted movement speed of 50 not sure if they need more but will compare against vid.

**Review by KnowOne134 — COMMENTED** [review]
we sure all the mobs in that family are movement of 75 before i say merge? i went to my database and filtered all pools that had family 266 and pulled up 30 pools, such as Vouivre, Pyrodrake, Hurricane_Wyvern, Bune. to name a few

**Comment by 59blargedy** [comment]
I checked: riverne, ifrit, gustav, abyssea-grau nms and standard mobs are
75 per npc database. If we need all 30 pools I can go validate each one if
preferred.

**Review by KnowOne134 — APPROVED** [review]
Awaiting 2nd approval

File: sql/mob_skill_lists.sql
Comment (ibm2431) [inline sql/mob_skill_lists.sql]: Make sure the name matches the mob name (`Lightning_Wyvern`), and sort the skill list ID (1011) to be above the Chaos Wyvern (1012) and I'll hit the button.

File: scripts/zones/Chamber_of_Oracles/mobs/Blizzard_Wyvern.lua
Comment (ibm2431) [inline scripts/zones/Chamber_of_Oracles/mobs/Blizzard_Wyvern.lua]: Set mod in `onMobFight` or `onMobEngaged`?

**Review by ibm2431 — CHANGES_REQUESTED** [review]

File: scripts/zones/Chamber_of_Oracles/mobs/Blizzard_Wyvern.lua
Comment (59blargedy) [reply scripts/zones/Chamber_of_Oracles/mobs/Blizzard_Wyvern.lua]: I'm not sure which is right - they basically use it as soon as they see you. if onmobfight is better, will do there. it works as is in terms of testing, but rather do it in the right place if they are different.

Comment (ibm2431) [reply scripts/zones/Chamber_of_Oracles/mobs/Blizzard_Wyvern.lua]: onMobEngaged is when the mob aggros (and also when it reaggros); onMobFight is constantly - I believe it's every attack round

Comment (59blargedy) [reply scripts/zones/Chamber_of_Oracles/mobs/Blizzard_Wyvern.lua]: oh yes, i remember now. most definitely how i have it, when it aggros in onmobengaged

**Review by ibm2431 — APPROVED** [review]

**Pull request merged**
